src/components/ProjectListItemCard/ProjectListItemCard.tsx

```diff
@@ -107,13 +107,8 @@ export const ProjectListItemCard: FC<ProjectListItemCardProps> = ({
             </Typography>
           </div>
           <div className={rightContent}>
-            <Typography variant="h1">
-              {0}{' '}
-              {/* Should be updated with implementation of engagements for project. Graphql query is also pending for this. */}
-            </Typography>
+            <Typography variant="h1">{props.engagements.total}</Typography>
             <Typography variant="body2" color="primary" align="right">
-              Languages
-              <br />
               Engagements
```

Comment:
We actually need to display "Language Engagements" or "Internship Engagements" based on the `type`.